src/__tests__/unit/middleware.test.ts

```diff
@@ -133,6 +129,27 @@ describe('middleware', () => {
     }
   });
 
+  it(`sets a cookie and redirects the user to the same route without the 
+  inviteCode search parameter when the inviteCode search parameter is detected.`, async () => {
+    const route = '/share';
+    const inviteCode = createId();
+    const fullPath = `${host}/${route}?${SearchParams.InviteCode}=${inviteCode}`;
```

Comment:
Because route = '/share' this is creating a double slash: https://challenge.8by8.us//share. Not a big deal, but not sure this is what is intended

Reply:
Nope! That's a mistake, thank you for catching it! I will fix it :)

Reply:
Fixed!